cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala

```diff
@@ -174,8 +182,9 @@ class AggregatingFunctionsTest extends DocumentingTest {
       }
     }
     section("min()", "functions-min") {
-      p("`min()` takes a numeric property as input, and returns the lowest value in that column.")
-      function("min(expression)", ("expression", "A numeric expression."))
+      p("`min()` returns the minimum value in a set of values.")
+      function("min(expression)", "Either an Integer or a Float or a String, depending on the values returned by `expression`.", ("expression", "An expression returning a set of numeric or string values."))
+      considerations("Any `null` values are excluded from the calculation.", "`min(null)` returns `null`.")
```

Comment:
The example is not really helpful since you can't see the data model or sample data! This is applicable to a fair number of the examples in this chapter (e.g. 3.4.2.1 (coalesce()) and 3.4.2.10 (size()). In some other examples, e.g. the one in 3.4.2.7 properties(), the data is created _within_ the example so the result is illustrative. Similarly in 3.4.2.15 toInteger(), the example is very clear and easy to understand.